### PR TITLE
Zarr

### PR DESCRIPTION
**Pull request by mfranzon**
Add scripts for:
1. creation of a zarr file on the local fs, then push on an s3 bucket.
2. creation of a zarr file on an s3 bucket.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+
+services:
+  localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME:-localstack-main}"
+    image: localstack/localstack
+    ports:
+      - "127.0.0.1:4566:4566"            # LocalStack Gateway
```
Comment (tcompa) [inline docker-compose.yml:8]: Should this be 0.0.0.0? @mfranzon

File: test_zarr_from_local_fs.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            write_image(ones((1, 1, 1, 256, 256)), image)
+    print(f"Created a zarr file here: {file_path}")
+    
+
+start_time_write = time.perf_counter()
+write_HCS_plate(local_path)
+end_time_write = time.perf_counter()
+elapsed_time_write = end_time_write - start_time_write
+
+print("Elapsed time for writing on localfs: ", elapsed_time_write)
```
Comment (tcompa) [inline test_zarr_from_local_fs.py]: What are we profiling here? :)

File: docker-compose.yml (hunk 1 above)
Comment (mfranzon) [reply docker-compose.yml:8]: no, there is no more UI in docker image

File: test_zarr_from_local_fs.py (hunk 2 above)
Comment (mfranzon) [reply test_zarr_from_local_fs.py]: time for creating the same HCS plate on localfs, end_time should be move below after the push on s3

**Pull request merged**